docs/docs/client-source-without-graphql.md

```diff
@@ -0,0 +1,184 @@
+---
+
+## title: "Client-side sourcing with JSON or YAML"
```

Comment:
Three things need to change here:
1.  This is happening at build time, so a better title would be something like "Sourcing Content from JSON or YAML" under "Sourcing Content and Data" (see my review for more on this)
2. Doc headlines need to live in frontmatter
3. The title should match the filename / URL
```
---
title: "Sourcing Content from JSON or YAML"
---
```

Reply:
@marcysutton regarding items 1 and 2 i mentioned before when answering @pieh that something went wrong with the linting job. My fork locally was updated and well as the dependencies before creating the pull request. Running `npm run format:other` originated that misconfiguration in the markdown files. Regarding item 3 it's been corrected and will be committed as soon as i finalize the changes you requested.